notanorm/model.py

```diff
@@ -93,6 +93,7 @@ class DbIndex(NamedTuple):
     fields: Tuple[DbIndexField, ...]
     unique: bool = False  # has a unique index?
     primary: bool = False  # is the primary key?
+    ix__name: str = None  # only avail when reading from a db model, not a ddl
```

Comment:
technically Optional[str] if it can be None

also, why use the double underscore?

Reply:
so nobody uses it